docs/release-notes/8.4.asciidoc

```diff
@@ -9,6 +9,41 @@
 [[known-issue-8.4.2]]
 ==== Known issues
 * A new Lucene 9 validation change may cause event correlation rule (EQL) errors whenever rule queries contain regular expressions using wildcard fields and predefined character classes (for example, `\w`, `\s`, `\d`).
+* In some situations, {elastic-endpoint} might change to a non-running state on Windows endpoints and cannot be restarted. This failure might be also missed by {agent}, which would automatically re-install {elastic-endpoint} (https://github.com/elastic/endpoint/issues/29[#29]).
++
+To determine whether {elastic-endpoint} has stopped running, run the following PowerShell command as an administrator:
```

Comment:
```suggestion
To determine whether {elastic-endpoint} has stopped running because of this issue, run the following PowerShell command as an administrator:
```